docs/events.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Event Subscription in Taquito
+id: subscribe_event
+author: Davis Sawali
+---
+
+# Contract Event Logging
+
+## Introduction
+Contract events is a way for contracts to deliver event-like information to third-party (off-chain) applications. It can be emitted by using the EMIT instruction in Michelson. 
+
+For more details and examples of how the EMIT instruction works, refer to [this article](https://tezos.gitlab.io/kathmandu/event.html).
+
+## Subscribing to Events in Taquito
+Taquito provides a simple way for users to subscribe to certain events on the blockchain via the `PollingSubscribeProvider`.
+
+### Example
+#### Usage
+```typescript
+const Tezos = new TezosToolkit(RPC_URL);
+
+Tezos.setStreamProvider(
+  Tezos.getFactory(PollingSubscribeProvider)({
+    shouldObservableSubscriptionRetry: true, 
+    pollingIntervalMilliseconds: 15000 
```

Comment:
A polling interval of 15000 is high and might leads to skipping some blocks. 
By default, the interval will be set using the RPC getConstants. We don't necessarily need to override it in the example.